gpt_index/vector_stores/types.py

```diff
@@ -34,6 +35,28 @@ class VectorStoreQueryResult:
     ids: Optional[List[str]] = None
 
 
+class VectorStoreQueryMode(str, Enum):
+    """Vector store query mode."""
+
+    DEFAULT = "default"
```

Comment:
nit: is `DEFAULT` always `DENSE` for all vector stores? If so, could we be explicit and use `DENSE`